### DISCUSSION TITLE
Programming Language Tag Dropdown

### DISCUSSION THREAD

**Discussion by 0xRenegade** [codinasion/codinasion-programme#1338] [Ideas]
There's already functionality to click the language you can see tagged to a Programme, but I feel that having a dropdown to the right of the "List of Programmes" title would help filter to your desired language, even if it's not on the first page.

**Comment by harshraj8843** [comment]
Thanks @0xRenegade for the suggestion 👍🏻 

This feature will be added soon 🚀 

This feature will be tracked in https://github.com/orgs/codinasion/projects/8
